app/models/case/sar/base_sar.rb

```diff
@@ -1,4 +1,6 @@
-class Case::SAR < Case
+class Case::SAR::BaseSAR < Case
+
+  # self.abstract_class = true
```

Comment:
Can get rid of this.

Reply:
NO - actually, it should go back in.

Reply:
Ah for some reason it doesn't work when you take have it in. Feels like we should have some check to make sure it's never saved.  I'll think about that later.